test-clients/src/main/java/com/hedera/services/legacy/regression/SmartContractTestInlineAssembly.java

```diff
@@ -56,6 +56,7 @@
 import java.security.PrivateKey;
```

Comment:
I also converted this test case completely to new EET based test case, no need to keep this one either.